### ISSUE TITLE
The URL must be of scheme file

### ISSUE THREAD

**Issue by phk422** [drwpow/vite-plugin-lqip#6]
## Description

Plugin error：The URL must be of scheme file

```
18:35:54 [vite] Internal server error: The URL must be of scheme file
      at new NodeError (node:internal/errors:399:5)
      at fileURLToPath (node:internal/url:1344:11)
      at Context.load (file:///D:/%E6%BA%90%E7%A0%81%E5%AD%A6%E4%B9%A0/vite-plugin-lqip/example/vite.config.ts.timestamp-1693910148240-d9aa0b54754fb.mjs:24:25)
      at Object.load (file:///D:/%E6%BA%90%E7%A0%81%E5%AD%A6%E4%B9%A0/vite-plugin-lqip/example/node_modules/.pnpm/vite@4.4.9/node_modules/vite/dist/node/chunks/dep-df561101.js:44254:64)
      at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
      at async loadAndTransform (file:///D:/%E6%BA%90%E7%A0%81%E5%AD%A6%E4%B9%A0/vite-plugin-lqip/example/node_modules/.pnpm/vite@4.4.9/node_modules/vite/dist/node/chunks/dep-df561101.js:54878:24)
      at async viteTransformMiddleware (file:///D:/%E6%BA%90%E7%A0%81%E5%AD%A6%E4%B9%A0/vite-plugin-lqip/example/node_modules/.pnpm/vite@4.4.9/node_modules/vite/dist/node/chunks/dep-df561101.js:64345:32)
```

| Name         | Version                      |
| :----------- | :--------------------------- |
| Node.js      | `20.0.0`                      |
| OS + version | ` Windows 10` |

## Reproduction

pnpm dev

**Comment by drwpow** [comment]
Fixed by #7

**Comment by drwpow** [comment]
This shipped in `v0.0.3`! Thank you!
